parsely/src/main/java/com/parsely/parselyandroid/UpdateEngagementIntervalCalculator.java

```diff
@@ -0,0 +1,23 @@
+package com.parsely.parselyandroid;
+
+import androidx.annotation.NonNull;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+class UpdateEngagementIntervalCalculator {
```

Comment:
Suggestion (💡): Ah, I forgot to mention this in my review, what about making this a `Utils` class instead of instantiating it as an object. Maybe even a Kotlin utils class, wdyt? 🤔

Reply:
In next iterations, I'll inject some `Clock` to this class to fake the current time. Otherwise, test for this class will be time-sensitive. So I'll have to leave this as a class, not a static one.

Reply:
Nice to know, thanks for the heads-up on that. 💯